### PR TITLE
PRIME-1124 Agreement Refactor/Rename

### PR DESCRIPTION
**Pull request by james-hollinger**
A bunch of renaming from Access Term -> Agreement.
Removed the DB view for newest agreement (your welcome, @itchison )
Other refactors.

### CODE REVIEW

File: prime-dotnet-webapi/Configuration/AgreementVersionConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                    new AgreementVersion { Id = 8,  AgreementType = AgreementType.RegulatedUserTOA,              Text =  "ru-access-terms-v4.html", EffectiveDate = DateTimeOffset.Parse("2020-06-03 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 9,  AgreementType = AgreementType.CommunityPharmacistTOA,        Text =  "com-pharm-terms-v1.html", EffectiveDate = SEEDING_DATE,                                CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 10, AgreementType = AgreementType.CommunityPharmacistTOA,        Text =  "com-pharm-terms-v2.html", EffectiveDate = DateTimeOffset.Parse("2020-08-28 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 11, AgreementType = AgreementType.CommunityPracticeOrgAgreement, Text = "com-practice-org-v1.html", EffectiveDate = SEEDING_DATE,                                CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
```
Comment (AnaisHebert) [inline prime-dotnet-webapi/Configuration/AgreementVersionConfiguration.cs:28]: We need to add the Community Pharmacist Org Agreement as well.

Comment (james-hollinger) [reply prime-dotnet-webapi/Configuration/AgreementVersionConfiguration.cs:28]: 💯

**Review by mtpultz — APPROVED** [review]
image

**Review by AnaisHebert — APPROVED** [review]
🍻🍻🍻🍻🍻🍻🍻🍻🍻🍻 Woo looks great

**Review by mtpultz — APPROVED** [review]
image

**Pull request merged**